diff --git a/straight.el b/straight.el
index 81ef345..05e3129 100644
--- a/straight.el
+++ b/straight.el
@@ -7320,6 +7320,68 @@ locally bound plist, straight-bug-report-args."
         (message "Dependents of %S: %S" package dependents)
       (nreverse dependents))))
 
+;;;; Default configuration
+
+(straight--reset-caches)
+
+(setq straight-recipe-repositories nil)
+
+(straight-use-recipes '(org-elpa :local-repo nil))
+
+(straight-use-recipes '(melpa :type git :host github
+                              :repo "melpa/melpa"
+                              :build nil))
+
+(if straight-recipes-gnu-elpa-use-mirror
+    (straight-use-recipes
+     '(gnu-elpa-mirror :type git :host github
+                       :repo "emacs-straight/gnu-elpa-mirror"
+                       :build nil))
+  (straight-use-recipes `(gnu-elpa :type git
+                                   :repo ,straight-recipes-gnu-elpa-url
+                                   :local-repo "elpa"
+                                   :build nil)))
+
+(straight-use-recipes
+ '(nongnu-elpa :type git
+               :repo "https://git.savannah.gnu.org/git/emacs/nongnu.git"
+               :depth (full single-branch)
+               :local-repo "nongnu-elpa"
+               :build nil))
+
+(straight-use-recipes '(el-get :type git :host github
+                               :repo "dimitri/el-get"
+                               :build nil))
+
+(if straight-recipes-emacsmirror-use-mirror
+    (straight-use-recipes
+     '(emacsmirror-mirror :type git :host github
+                          :repo "emacs-straight/emacsmirror-mirror"
+                          :build nil))
+  (straight-use-recipes '(emacsmirror :type git :host github
+                                      :repo "emacsmirror/epkgs"
+                                      :nonrecursive t
+                                      :build nil)))
+
+(if (straight--modifications 'check-on-save)
+    (straight-live-modifications-mode +1)
+  (straight-live-modifications-mode -1))
+
+(when (straight--modifications 'watch-files)
+  (straight-watcher-start))
+
+(if straight-use-symlinks
+    (straight-symlink-emulation-mode -1)
+  (straight-symlink-emulation-mode +1))
+
+(if straight-enable-package-integration
+    (straight-package-neutering-mode +1)
+  (straight-package-neutering-mode -1))
+
+(if straight-enable-use-package-integration
+    (straight-use-package-mode +1)
+  (straight-use-package-mode -1))
+
 ;;;; Closing remarks
 
 (provide 'straight)
